bridge/node/Cargo.toml

```diff
@@ -11,9 +11,9 @@ name = "dsnp_graph_sdk_node"
 crate-type = ["cdylib"]
 
 [dependencies]
-dsnp-graph-core = { version = "1.0.0", path = "../../core" }
-dsnp-graph-config = { version = "1.0.0", path = "../../config" }
-neon = { version = "0.10.1", default-features = false, features = ["napi-6"] }
-serde = { version = "1.0", features = ["derive"] }
-serde_json = "1.0.96"
-once_cell = "1.18.0"
+dsnp-graph-core = { version = "1.0.2", path = "../../core" }
+dsnp-graph-config = { version = "1.0.2", path = "../../config" }
+neon = { version = "1.0.0", default-features = false, features = ["napi-6"] }
```

Comment:
Cool, 1.0. Any significant enhancements to neon? Worth filing an issue in the backlog to review our usage?

Reply:
have not looked into it yet. Feel free to take a look since you guys  know more about it than me